src/citrine/resources/project.py

```diff
@@ -317,6 +317,8 @@ def pull_in_resource(self, *, resource: Resource):
         return True
 
     @use_teams("project.publish")
```

Comment:
Is this decorator already warning folks to use `project.publish` instead?  Trying to understand the interaction here.